test/common/access_log/test_util.h

```diff
@@ -156,6 +157,11 @@ class TestRequestInfo : public RequestInfo::RequestInfo {
     (*metadata_.mutable_filter_metadata())[name].MergeFrom(value);
   };
 
+  const ::Envoy::RequestInfo::FilterState& perRequestState() const override {
```

Comment:
Missed these.

Reply:
Please remove the `::Envoy::` prefix.

Reply:
Huh.  Coudla sworn that qualifier wasn't there when I was looking at your comment originally, but that doesn't make sense since github binds comments to hashes.  But removing them won't work--the dependent files won't then compile:

ERROR: /usr/local/google/home/rdsmith/Checkouts/envoy/test/common/access_log/BUIL\
D:31:1: C++ compilation of rule '//test/common/access_log:access_log_formatter_fu\
zz_test_lib' failed (Exit 1)
In file included from ./test/fuzz/utility.h:5:0,
                 from test/common/access_log/access_log_formatter_fuzz_test.cc:5:
./test/common/access_log/test_util.h:160:22: error: 'FilterState' in 'class Envoy\
::RequestInfo::RequestInfo' does not name a type
   const RequestInfo::FilterState& perRequestState() const override {
                      ^~~~~~~~~~~
./test/common/access_log/test_util.h:163:16: error: 'FilterState' in 'class Envoy\
::RequestInfo::RequestInfo' does not name a type
   RequestInfo::FilterState& perRequestState() override { return per_request_stat\
e_; }
                ^~~~~~~~~~~
./test/common/access_log/test_util.h:187:16: error: 'FilterStateImpl' in 'class E\
nvoy::RequestInfo::RequestInfo' does not name a type
   RequestInfo::FilterStateImpl per_request_state_{};
                ^~~~~~~~~~~~~~~

(While I hadn't understood your comment, I had grepped for ::Envoy in my diff and removed them all, then re-added the ones that caused compilation errors.)

Reply:
Oh I see, it's because RequestInfo here refers to the base class. Can drop the leading colons (e.g., Envoy::RequestInfo::FilterState) to match the other references to the RequestInfo namespace in this file.

Reply:
Ah, didn't think to try that.  Done.